overlord/snapstate/snapstate_test.go

```diff
@@ -2689,7 +2719,7 @@ version: 1.0`)
 		{
 			// ensure only local install was run, i.e. first action is pseudo-action current
 			op:  "current",
-			old: "/snap/mock/100001",
+			old: filepath.Join(dirs.StripRootDir(dirs.SnapMountDir), "mock/100001"),
```

Comment:
This feels a bit repetitive and relatively long, I wonder if we could compute the dirs only once at the beginning of the test. OTOH it was repetitive before so maybe that is something for a later branch.